praw/__init__.py

```diff
@@ -380,9 +389,12 @@ def handle_redirect():
                 request.url = url
                 kwargs['_cache_key'] = (normalize_url(request.url),
                                         tuple(key_items))
-                response = self.handler.request(request=request.prepare(),
-                                                proxies=self.http.proxies,
-                                                timeout=timeout, **kwargs)
+                response = \
+                    self.handler.request(request=request.prepare(),
+                                         proxies=self.http.proxies,
+                                         timeout=timeout,
+                                         verify=self.http.validate_certs,
```

Comment:
for the verify -> validate_certs naming issue.. I'm open to suggestions for renaming "validate_certs" but felt that "verify" was too ambiguous for the ini file.